packages/plugin-auto-tracking-browser/src/helpers.ts

```diff
@@ -51,3 +51,45 @@ export const getText = (element: Element): string => {
   }
   return text;
 };
+
+export const isPageUrlAllowed = (url: string, pageUrlAllowlist: string[] | undefined) => {
```

Comment:
Is this matched against full url or path names? As written I would expect to have to define entire URLs and not just a list of paths

Reply:
This is a regex match. As I think in most of the cases it would be a regex-like entry instead of a specific url, but it can be achievable as we support regex here. Example of allow list:
- `["https://amplitude.com/blog"]`, then `https://amplitude.com/blog/abc` should work here.
- `["http.?://*.amplitude"]`, then subdomains should work here.
- `["https://amplitude.com/blog$"]`, then should only match `https://amplitude.com/blog`.
I will add more tests in code and examples in the doc. (cc @justin-fiedler if you have any thoughts on this).

Reply:
@justin-fiedler , in this case, I can update the type to be a list of string or regex, so if it is a regex, it will work like above (user needs to provide a regex instead of string), if it is a string, then we do the exact match. How about this approach?

Reply:
Nice, I like that 👍